src/theme/components/tabs.ts

```diff
@@ -38,9 +37,25 @@ const tab = {
   _focus: {
     boxShadow: "0",
   },
+  _dark: {
+    color: "dark.ui.typography.heading",
+    bg: "dark.ui.bg.default",
+    border: "0",
```

Comment:
There is some minor shifting happening between the light and dark variants.  I was able to resolve the issue by changing this to the following, but I am not exactly sure why it is needed.
```
border: "1px solid transparent",
```